### PR TITLE
[Toolbar] Fix Back Arrow In Searchable

### PR DESCRIPTION
**Pull request by habibridho**
#182

### CODE REVIEW

File: src/Toolbar/LeftElement.react.js [hunk 1]
```diff
@@ -90,12 +90,12 @@ class LeftElement extends PureComponent {
             easing: Easing.linear,
             useNativeDriver: Platform.OS === 'android',
         }).start(() => {
-            let leftElement = activate ? SEARCH_FORWARD_ICON : this.props.leftElement;
+            const leftElement = activate ? SEARCH_FORWARD_ICON : this.props.leftElement;
 
-            if (!this.state.leftElement) {
```
Comment (chirag04) [inline src/Toolbar/LeftElement.react.js]: are you sure this is the right fix?

**Comment by janziemba** [comment]
Current design:
screen shot 2017-08-31 at 12 59 53

Closes #182

File: src/Toolbar/LeftElement.react.js (hunk 1 above)
Comment (xotahal) [reply src/Toolbar/LeftElement.react.js]: @habibridho Why it should fix this problem?

**Comment by xotahal** [comment]
Closing this, since I don't think it will help. But feel free to open it again ;)

**Comment by janziemba** [comment]
Reopened my old duplicate PR: https://github.com/xotahal/react-native-material-ui/pull/196

**Pull request closed without merging**